src/group_theory/subgroup.lean

```diff
@@ -779,6 +779,16 @@ lemma mem_map {f : G →* N} {K : subgroup G} {y : N} :
   y ∈ K.map f ↔ ∃ x ∈ K, f x = y :=
 mem_image_iff_bex
 
+@[to_additive]
+lemma mem_map_of_mem {K : subgroup G} (f : G →* N) (x : K) :
+  f x ∈ K.map f :=
+mem_image_of_mem f x.prop
+
+@[to_additive]
+lemma mem_map_of_mem' {K : subgroup G} (f : G →* N) {x : G} (hx : x ∈ K) :
+  f x ∈ K.map f :=
+mem_image_of_mem f hx
```

Comment:
To elaborate on my PR comment:
```suggestion
@[to_additive]
lemma mem_map_of_mem {K : subgroup G} (f : G →* N) {x : G} (hx : x ∈ K) :
  f x ∈ K.map f :=
mem_image_of_mem f hx

@[to_additive]
lemma apply_coe_mem_map {K : subgroup G} (f : G →* N) (x : K) :
  f x ∈ K.map f :=
mem_map_of_mem f x.prop
```

Reply:
Now the versions for `submonoid`, `subgroup` and `submodule` match. Let's see if this compile, I made some variable implicit, following `submodule.mem_map_of_mem`.